calico/felix/fetcd.py

```diff
@@ -129,6 +149,67 @@ def _periodically_resync(self):
             gevent.sleep(sleep_time)
             self.force_resync(reason="periodic resync", async=True)
 
+    @logging_exceptions
+    def _status_reporting(self):
+        """
+        Greenlet: periodically writes to etcd
+
+        :return: Does not return, unless reporting disabled.
+        """
+        _log.info("Started status reporting thread.")
+        ttl = self._config.REPORTING_TTL_SECS
+        interval = self._config.REPORTING_INTERVAL_SECS
+        _log.info("Config loaded, reporting interval: %s, TTL: %s", interval, ttl)
+        if interval == 0:
```

Comment:
A blank line before and after this if block would break up the code into logical sections and make it more readable.